release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -63,6 +63,11 @@ You can now install a cluster on Google Cloud Platform (GCP) and use a personal
 [id="ocp-4-7-web-console"]
 === Web console
 
+[id="ocp-4-7-web-console-localization"]
+==== Web console localization
+
+The web console is now localized and provides language support for global users. English, Japanese, Simplified Chinese, and Korean are currently supported. The displayed language follows your browser preferences, but you can also select a language to override the browser default. From the *Admin* drop-down menu in the *Administrator* console, select *Language preferences* to update your language setting. Localized date and time is now also supported.
```

Comment:
How about update "From the *Admin* drop-down menu in the *Administrator* console" to "From the  *Admin* drop-down menu on the top right corner"?  Since the dropdown menu are clickable no matter in "Administrator" console or "Developer" console.